app/src/main/java/org/wikipedia/readinglist/ReadingListBehaviorsUtil.kt

```diff
@@ -162,8 +162,8 @@ object ReadingListBehaviorsUtil {
             return
         }
         FeedbackUtil
-                .makeSnackbar(activity, if (pages.size == 1) activity.getString(R.string.reading_list_item_deleted, pages[0].title())
-                                else activity.resources.getQuantityString(R.plurals.reading_list_articles_deleted, pages.size, pages.size),
+                .makeSnackbar(activity, if (pages.size == 1) activity.getString(R.string.reading_list_item_deleted_from_list, pages[0].title(), readingList.title())
+                else activity.resources.getQuantityString(R.plurals.reading_list_articles_deleted_from_list, pages.size, pages.size, readingList.title()),
```

Comment:
Looks like the first string item has two parameters instead of one?

Reply:
It has 2 parameters @cooltey